templates/base.html

```diff
@@ -88,17 +88,21 @@
                             {% if user.is_authenticated %}
                             {% group_select %}
                             {% notification_menu %}
-                            <li role="presentation" class="menu-profile">
-                                <a href="{% url 'profile' %}">
-                                    <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
-                                    {{ user.username }}
-                                </a>
-                            </li>
-                            <li role="presentation">
-                                <a href="{% url 'logout' %}">
+                            <li class="dropdown">
+                              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                                <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
+                                {{ user.first_name }} {{ user.last_name }} <span class="caret"></span>
```

Comment:
Maybe one of these would be cleaner: `user.get_full_name` (in django AbstractUser) or `user.userprofile.shortname` (in a-plus/userprofile/models.py).

Reply:
rofl.. the `shortname` is currently used only in some tests... so maybe go with the former.

Reply:
I don't know what I'm thinking. It's probably better to have this here. so I'll just merge this.